mne/io/eeglab/eeglab.py

```diff
@@ -717,15 +717,15 @@ def _read_annotations_eeglab(eeg):
         description = []
     elif isinstance(eeg.event, np.ndarray):
         description = [str(event.type) for event in eeg.event]
-        onset = [event.latency for event in eeg.event]
+        onset = [event.latency - 1 for event in eeg.event]
```

Comment:
Maybe add a comment here, just to be sure?

Reply:
see a comment by @agramfort, few lines below, I'll leave it as it is for now.